toolchain/binutils.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
Actually you can inspire here: https://github.com/avocado-framework/avocado-misc-tests/pull/26 for heading (both versions are acceptable)